### ISSUE TITLE
Update external site information regarding ampache

### ISSUE THREAD

**Issue by lachlan-00** [ampache/ampache#1882]
There is a lot of out of date information about Ampache on the internet.

After a quick google the following sites could do with some love.

https://en.m.wikipedia.org/wiki/Ampache
https://help.ubuntu.com/community/ampache
https://wiki.archlinux.org/index.php/Ampache
https://wiki.debian.org/Ampache
https://alternativeto.net/software/ampache

**Comment by kuzi-moto** [comment]
Good idea! I think having up-to-date information everywhere about Ampache would give a better impression, and would let people know it's still being worked on. I was going to ask, is the Ampache squirrel still technically the mascot? I don't see any references to it anywhere on ampache.org nor anywhere in the repo.

**Comment by lachlan-00** [comment]
I actually think the reborn logo is the official logo?

![](https://ampache.lachlandewaard.org/themes/reborn/images/ampache-dark.png)

It's actually been the main logo on the software for so long that i think we should make it official.

**Comment by kuzi-moto** [comment]
Just as a thought, would also be nice to update the various Linux distribution's packages as well. I think this would be quite an effort though, and I know almost nothing about creating/updating Linux packages. But something to think about in the future. I think anything to help spread Ampache's reach as well as improve its popularity is good! Here's a nice site I found with a list of various distributions and their repositories, seems up to date: https://repology.org/project/ampache/packages

**Comment by kuzi-moto** [comment]
Made some progress on Wikipedia, but could use some more work.

**Comment by AshotN** [comment]
I don't know if anyone knows the guy who is the sole moderator of [/r/ampache](https://www.reddit.com/r/ampache/) 

But we can try to revive the sub.

**Comment by AshotN** [comment]
> I actually think the reborn logo is the official logo?
> 
> ![](https://camo.githubusercontent.com/626581dcd2068c13df7e2da62b4943a8115faf23/68747470733a2f2f616d70616368652e6c6163686c616e646577616172642e6f72672f7468656d65732f7265626f726e2f696d616765732f616d70616368652d6461726b2e706e67)
> 
> It's actually been the main logo on the software for so long that i think we should make it official.

Also your image link is dead for whatever reason. 

> https://wiki.debian.org/Ampache

The [Debian package](https://packages.debian.org/jessie/ampache) for Ampache is also severely outdated. I don't know if anyone here knows how to get that updated.

**Comment by lachlan-00** [comment]
It was probably dropped from debian for inactivity. It's been missing for the last 2 stables.
Someone within debian would have to want to maintain it or we would have to apply to debian
